configurator/parsers.py

```diff
@@ -12,6 +10,9 @@ class Parsers(dict):
     
     @classmethod
     def from_entrypoints(cls):
+        # `pkg_resources` is slow to import, so defer until we need it.
+        from pkg_resources import iter_entry_points
```

Comment:
This is a bit ugly. Why is important `pkg_resources` so slow? Have you reported the issue upstream at https://github.com/pypa/setuptools?

Reply:
It is a bit ugly.  I don't know why it's so slow, but there are already a couple issues open relating to this: pypa/setuptools#510 pypa/setuptools#926.